tests/controller_tests/test_ingest.py

```diff
@@ -29,71 +27,57 @@
     Project,
     Submission,
 )
-from core.exceptions import ValidationError
-from core.validation.exceptions import UnimplementedErrorMessageException
-from core.validation.failures.user import (
-    NonNullableConstraintFailure,
-    WrongInputFailure,
-)
 
 resources = Path(__file__).parent / "resources"
 
 
-@pytest.fixture
-def wrong_format_test_file() -> BinaryIO:
-    """An invalid text test file."""
-    with open(resources / "wrong_format_test_file.txt", "rb") as file:
-        yield file
-
-
-"""
-/ingest endpoint validation_tests
-"""
-
-
-def test_ingest_endpoint(test_client_reset, example_data_model_file, mocker):
-    """Tests that, given valid inputs, the endpoint responds successfully."""
-    load_data_mock = mocker.spy(ingest, "load_data")
-    endpoint = "/ingest"
-    response = test_client_reset.post(
-        endpoint,
-        data={
-            "excel_file": example_data_model_file,
-        },
-    )
-
-    assert load_data_mock.called
-    assert response.status_code == 200, f"{response.json}"
-    assert response.json == {
-        "detail": "Spreadsheet successfully validated and ingested",
-        "metadata": {},
-        "status": 200,
-        "title": "success",
-        "loaded": True,
-    }
-
+def example_table_data_to_be_loaded_via_populate_db():
+    """
+    Helper function that ingests example data via populate_db() in order to be loaded into the database.
```

Comment:
Can we make this a `fixture` with module level scope and then we only need to do it once. The tests can then assert on the db contents?

Although I wonder if the fact that `populate_db` commits should be a consideration

Reply:
Ah yeah I see. Yep a fixture that returns the `data_dict` etc would be ideal. Then we can call `populate_db` in the tests directly, just passing the `mock_r3_data_dict` etc from the fixture